dashboard/test/ui/utils/selenium_browser.rb

```diff
@@ -3,13 +3,13 @@
 module SeleniumBrowser
   def self.webdriver_options_object(browser: :chrome, headless: false)
     options = Selenium::WebDriver::Options.send(browser)
-    options.add_argument('window-size=1280,1024') if [:chrome, :firefox].include?(browser)
-    options.add_argument('headless') if headless
+    options.add_argument('--window-size=1280,1024') if [:chrome, :firefox].include?(browser)
+    options.add_argument('--headless') if headless
```

Comment:
Not part of the review, just me being curious: were these not taking effect before without the -- being added?

Reply:
They were working for Chrome, which I think is what most people use for manual local testing, but not for Firefox